net.pioneerspacesim.Pioneer.json

```diff
@@ -0,0 +1,102 @@
+{
+	"app-id": "net.pioneerspacesim.Pioneer",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "1.6",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "pioneer",
+
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--device=dri",
+		"--persist=.pioneer"
+	],
+
+	"build-options" : {
+		"cflags": "-O3",
+		"cxxflags": "-O3",
+		"strip": true,
```

Comment:
Debug symbols are extracted into extension `appid.Debug` so makes no sense to disable it as it helps with debugging.